File: part-c-calculator/calculator.c
#include <stdio.h>

int main() {
    double num1, num2;

    printf("Enter first number: ");
    scanf("%lf", &num1);

    printf("Enter second number: ");
    scanf("%lf", &num2);

    // scanf converts the input into numbers.
    printf("Sum: %f\n", num1 + num2);
    printf("Difference: %f\n", num1 - num2);
    printf("Product: %f\n", num1 * num2);
    printf("Quotient: %f\n", num1 / num2);

    return 0;
}

File: part-c-calculator/calculator.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter first number: ", function(first) {
    rl.question("Enter second number: ", function(second) {
        // Number() converts the input text into numbers.
        let num1 = Number(first);
        let num2 = Number(second);

        console.log("Sum:", num1 + num2);
        console.log("Difference:", num1 - num2);
        console.log("Product:", num1 * num2);
        console.log("Quotient:", num1 / num2);

        rl.close();
    });
});